src/utils/selections.js

```diff
@@ -22,13 +22,14 @@ export default {
       const activate = ind === -1;
       let linkedIds = [];
 
-      if (linked) {
+      if (!singleSelect) {
         linkedIds = getAllTreeElemNo(node, activate);
       }
       if (!activate) {
         newState.splice(ind, 1);
         linkedIds.forEach(id => {
-          newState.splice(newState.indexOf(id), 1);
+          const idInd = newState.indexOf(id);
+          idInd !== -1 && newState.splice(idInd, 1);
```

Comment:
This is actually to fix a bug that Mirja found. When deselecting from chained selection we removed other selections in the array